site/bin/_lib.sh

```diff
@@ -258,14 +281,17 @@ release() {
   mv "${target}/docs/index-release.md" "${target}/docs/index.md"
 
   echo "     ... replace title in versioned mkdocs.yml"
-  sed -i'' -E "s/(^site_name:[[:space:]]+).*$/\1\"Nessie ${version}\"/" "${target}/mkdocs.yml"
+  sed -i'' -E "s/(^site_name:[[:space:]]+).*$/\1\"Nessie ${RELEASE_VERSION}\"/" "${target}/mkdocs.yml"
```

Comment:
This is actually a fix

Reply:
This is the same command as the one in line 166. Don't we need to distinguish Darwin vx Linux here too?

Reply:
Hmm I see; you are assuming this is always executed by Github CI as part of the release process.